.github/workflows/go-update-commenter.yml

```diff
@@ -0,0 +1,65 @@
+name: "Go update commenter"
+
+on:
+  pull_request:
+    # Only run on PR label events (in particular not on every commit)
+    types: [ labeled ]
+
+jobs:
+  old-versions-match:
+    # Only run if the PR is labeled with 'go-update'
+    if: ${{ github.event.label.name == 'go-update' }}
+    runs-on: ubuntu-latest
+    steps:
+      # get the Go version of the target branch
+      - uses: actions/checkout@v3
```

Comment:
Should we use the latest version of `actions/checkout@v4`?

Reply:
Seems like the repo uses v3 for the most part (only a single workflow uses v4) so I'd rather stick to that and let platform decide on when to do the update :smile: